cumulusci/tasks/robotframework/robot_lint.py

```diff
@@ -0,0 +1,219 @@
+from cumulusci.core.tasks import BaseTask
+from cumulusci.core.utils import process_bool_arg
+from cumulusci.core.utils import process_list_arg
+from cumulusci.core.exceptions import CumulusCIFailure, CumulusCIUsageError
+import rflint
+import glob
+import os
+
+
+class RobotLint(BaseTask):
+    salesforce_task = False  # does not require an org
+    task_docs = """
+    The robot_lint task performs static analysis on one or more .robot
+    and .resource files. Each line is parsed, and the result passed through
+    a series of rules. Rules can issue warnings or errors about each line.
+
+    If any errors are reported, the task will exit with a non-zero status
+
+    When a rule has been violated, a line will appear on the output in
+    the following format:
+
+    *<severity>*: *<line>*, *<character>*: *<description>* (*<name>*)
+
+    - *<severity>* will be either W for warning or E for error
+    - *<line>* is the line number where the rule was triggered
+    - *<character>* is the character where the rule was triggered,
+      or 0 if the rule applies to the whole line
+    - *<description>* is a short description of the issue
+    - *<name>* is the name of the rule that raised the issue
+
+    Note: the rule name can be used with the ignore, warning, error,
+    and configure options.
+
+    Some rules are configurable, and can be configured with the
+    `configure` option. This option takes a list of values in the form
+    *<rule>*:*<value>*,*<rule>*:*<value>*,etc.  For example, to set
+    the line length for the LineTooLong rule you can use '-o configure
+    LineTooLong:80'. If a rule is configurable, it will show the
+    configuration options in the documentation for that rule
+
+    The filename will be printed once before any errors or warnings
+    for that file. The filename is preceeded by `+ `
+
+    Example Output::
+
+        + example.robot
+        W: 2, 0: No suite documentation (RequireSuiteDocumentation)
+        E: 30, 0: No testcase documentation (RequireTestDocumentation)
+
+    To see a list of all configured options, set the 'list' option to True:
+
+        cci task run robot_list -o list True
+
+    """
```

Comment:
Looks like you need to run `cci task doc > docs/tasks.rst` to update the file which becomes the source of https://cumulusci.readthedocs.io/en/latest/tasks.html

Reply:
Cool. I didn' t know `cci task doc` was a thing.

Reply:
I should probably figure out how to make it run in a precommit hook.